### DISCUSSION TITLE
Personal respawn tickets reset

### DISCUSSION THREAD

**Discussion by ghost** [Vdauphin/HeartsAndMinds#1331] [Question and answer]
Good afternoon,

We started using the respawn tickets, unique to each player. We all started with 10 tickets, but after about 48 hours, some people (who hadn't died yet) only had one ticket and was forced into spectator mode.

Do we know if anything would cause tickets to reset to 1? 

Checked that params were set to individual tickets, not side and set to 10.

We did take a prisoner before this happened, I personally went up to 11 tickets. After logging back in the next day I saw (with Zeus interface) that I only had 1 ticket left.

Thanks!

**Comment by Vdauphin** [comment]
Hello,

> Do we know if anything would cause tickets to reset to 1?

May be just a wrongly edited .pbo or mod changing something

Good luck

**Reply by ghost** [reply]
I doubt it's mod, as it's just ACE / RHS stuff, but I'll check the files and see if I can find something.

Thanks

**Reply by ghost** [reply]
I'm going to test without my saving of player movement and loadout to profilenamespace...wonder if there was a crossover there.

Loving the mission and where you've taken it over the last few years @Vdauphin !

**Reply by ghost** [reply]
Looks like we are running into the same issue, with no mods changing steam ID's...

What's odd, is that it only effects some people, only one person has used the server, hasn't died, and went from 15 tickets to 0 after one death. 

Could it be an issue with Steam UID not initializing or something?

**Reply by Vdauphin** [reply]
> Looks like we are running into the same issue, with no mods changing steam ID's...

Well this is impossible for a mod to change steam ID

> What's odd, is that it only effects some people, only one person has used the server, hasn't died, and went from 15 tickets to 0 after one death.

You just have something setting 0 tickets with `BIS_fnc_respawnTickets` to this player when he died

> Could it be an issue with Steam UID not initializing or something?

Steam UID is not used by BI respawn

**Comment by Vdauphin** [comment]
Should be fix by this https://github.com/Vdauphin/HeartsAndMinds/pull/1360

Cheers
